web/src/buddy_list.ts

```diff
@@ -224,7 +250,7 @@ export class BuddyList extends BuddyListConf {
                                         defaultMessage:
                                             "{N, plural, one {# other subscriber} other {# other subscribers}}",
                                     },
-                                    {N: total_human_subscribers_count - participant_count},
+                                    {N: non_participant_subscribers_count},
                                 );
                             } else if (current_sub) {
```

Comment:
This can be a follow-up, but I wonder if we can use `non_participant_subscribers_count` all the other places in this function that we use `total_human_subscribers_count`? After all, if `participant_count` is 0, these should be the same. 

It would be more efficient to not compute both if they will have the same value in the only places we use the other one.

Reply:
Yeah great idea, I'll add that to my list for after this and #33578 get merged :)